### ISSUE TITLE
Acromioclavicular (AC) joint separation

### ISSUE THREAD

**Issue by MickeySegal** [obophenotype/human-phenotype-ontology#11007]
**Preferred term label:**
Acromioclavicular (AC) joint separation

**Synonyms**

**Definition (free text, please give PubMed ID)**
AC joint x-ray views including bilateral anteroposterior (AP) view, axillary lateral view, and/or Zanca view showing either superior, posterior, or inferior displacement of the distal clavicle in relation to the acromion at the AC joint

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
Acromioclavicular (AC) joint injury

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
@MickeySegal 
Adding new term: 
- ID: HP:6001338
- label: Acromioclavicular joint separation
- parents: [Abnormal clavicle morphology (HP:0000889)](https://hpo.jax.org/app/browse/term/HP:0000889)
- pmid: PMID:23242975
- def: An acromioclavicular (AC) joint joint separation, often called a shoulder separation, is a dislocation of the clavicle from the acromion. This feature is ascertainable radiographically by AC joint x-ray views including bilateral anteroposterior (AP) view, axillary lateral view, and/or Zanca view showing either superior, posterior, or inferior displacement of the distal clavicle in relation to the acromion at the AC joint.
- comment: The AC joint is a diarthrodial joint spanning from the medial articular facet of the acromion to the distal clavicle and links the shoulder girdle to the axial skeleton.
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/11007
- synonyms:
